config/routes.rb

```diff
@@ -82,6 +82,7 @@
       post '/' => 'users/sessions#create', as: :user_session
       get '/logout' => 'users/sessions#destroy', as: :destroy_user_session
       get '/active' => 'users/sessions#active'
+      get '/sessions/keepalive' => 'users/sessions#keepalive'
```

Comment:
this has side effects, so can we make it a POST?

Reply:
yep, very good call

Reply:
Ran into a snag with CSRF, but added in 2ba6e0f

Reply:
Awesome! thanks